### DISCUSSION TITLE
Can I transfer my character from the test server over to chromie craft?

### DISCUSSION THREAD

**Discussion by CalebHyde** [chromiecraft/chromiecraft#6586] [General]
When I first went to make my new character, the chromiecraft server threw a warning saying it wasn't allowing new characters at the moment. I proceeded to create a tauren druid on the test server. After playing on it for several hours I'm realizing no one else is on it.

Is it possible to transfer this character over to the chromiecraft rp server?

My username is Exzidian,
The character on the test server I want to transfer is named 'Drylx'

Thanks

**Comment by BeefSteaker** [comment]
Hi, unfortunately transfers from the ptr to live aren't possible due to the ptr having free access to many GM commands.

If you're getting a character creation error on live, it's likely you're trying to make a death Knight. These are disabled until we reach wrath content in progession

**Comment by CalebHyde** [comment]
ok, gotchya. That all makes sense.
Thanks for the snappy response.

I'll just fall in love with my druid being forever alone and lean into it. heh. 
Will be a fun solo server!

I'll make a new character or two on the live server next.
Cheers for now
